pkg/tnf/handlers/container/pod.go

```diff
@@ -100,7 +100,8 @@ func (p *Pod) ReelMatch(_, _, match string) *reel.Step {
 			p.result = tnf.SUCCESS
 			return nil
 		}
-		replacer := strings.NewReplacer(`[`, ``, "\"", ``, `]`, ``, `, `, `,`)
+		replacer := strings.NewReplacer(`[`, ``, "\"", ``, `]`, ``, `, `, `,`, "\n", ``, "\t", ``, ` `, ``)
```

Comment:
``, `, ` `` is no longer needed

Reply:
Actually, Gonzalo brought up a good point. Removing space this way is a bit too much. @shimritproj Let's remove both
````, `, `,` and ` `, ``
do this instead:
```suggestion
		if contains(p.ExpectStatus, strings.Trim(status, " ")) {
```

Reply:
@jun-rh  @greyerof right, I changed.